### ISSUE TITLE
Your boxes are baked...

### ISSUE THREAD

**Issue by PseudoRAM** [Kaleado/cwm-website-material-new#3]
.teamImg {
    height: 100%;
}

Are you purposely distorting your images on resize?

**Comment by PseudoRAM** [comment]
@Kaleado

**Comment by Kaleado** [comment]
Yes, I thought it looked better to resize the images. If you disagree,  feel free to change it.

**Comment by PseudoRAM** [comment]
@Kaleado If you're going to resize them I'd assume that you'd want to keep the image aspect ratio, otherwise if you stick in a team members portraits you'll have everyone looking like horses
